src/pages/limyujin/Main/Components/Feeds.js

```diff
@@ -0,0 +1,201 @@
+import React from 'react';
+import CommentColumn from './CommentColumn';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faEllipsisH } from '@fortawesome/free-solid-svg-icons';
+
+import {
+  faComment,
+  faPaperPlane,
+  faBookmark,
+  faSmile,
+} from '@fortawesome/free-regular-svg-icons';
+
+import './Feeds.scss';
+import './CircleProfile.scss';
+
+class Feeds extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      commentValue: '',
+      commentCounter: 0,
+      commentTexts: [],
+      isFeedLikePressed: false,
+    };
+  }
+
+  handleInput = e => {
+    this.setState({
+      commentValue: e.target.value,
+    });
+  };
+
+  handleSubmitBtn = e => {
+    e.preventDefault();
+
+    if (this.state.commentValue) {
+      const createComment = () => ({
+        id: this.state.commentCounter++,
+        text: this.state.commentValue,
+      });
+
+      this.setState({
+        commentTexts: [...this.state.commentTexts, createComment()],
```

Comment:
```suggestion
        commentTexts: [...this.state.commentTexts, {id: 어쩌구, text: 어쩌구}],
```
혹시 createComment함수 따로 안빼고 이렇게 해도 되는가 궁금합니다.

Reply:
확인해봤는데 잘 작동됩니다 !!